tox.ini

```diff
@@ -24,3 +24,10 @@ application-import-names = flake8
 select = B, C, E, F, W, B, B950
 import-order-style = pep8
 max-complexity = 10
+
```

Comment:
Hmm, if we're adding this testenv then I think we can dispense with the `testenv:linting` above, since they're doing the same thing (or should be).